great_ape_safe/ape_api/ebtc.py

```diff
@@ -177,7 +473,15 @@ def cancel_highsec_timelock(
     ##
     ##################################################################
 
-    def grant_timelock_role(self, role_key, account, use_high_sec=False):
+    def grant_timelock_role(
+        self, role_key, account, salt=EmptyBytes32, use_high_sec=False
+    ):
+        """
+        @dev Grants a role on the timelock to an account.
+        @param role_key The key of the role to grant. Can be one of "PROPOSER_ROLE", "CANCELLER_ROLE", "EXECUTOR_ROLE", or "TIMELOCK_ADMIN_ROLE".
+        @param account The account to grant the role to.
```

Comment:
seems missing `@param` for `salt`

Reply:
Thanks for the review. Good catch, also noticed this yesterday and addressed on the following PR: https://github.com/ebtc-protocol/ebtc-multisig/pull/17/commits/147550eb643bb5c405e729ecebdb34ca0abdfec7